modules/aos/scene/AosScene.cpp

```diff
@@ -584,12 +584,12 @@ namespace aos
         }
         else if(format_name == "class Omni::Geometry::Mesh::ply_serializer")
         {
-            //omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>* doc;
-            //doc = new omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>(store_key);
+            omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>* doc;
+            doc = new omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>(store_key);
             try
             {
-                mesh = GodotResolveMesh<Omni::Geometry::Mesh::ply>(store_key, false); // false for not binary
-                //mesh = doc->resolve_object().get();
+                //mesh = GodotResolveMesh<Omni::Geometry::Mesh::ply>(store_key, false); // false for not binary
```

Comment:
Remove dead code, and the method GodotResolveMesh can be removed as well.

Reply:
GodotResolveMesh can't be removed. It's still used.